example-plugins/charting_api_examples/routes/notes.py

```diff
@@ -0,0 +1,178 @@
+import arrow
+from http import HTTPStatus
+
+from canvas_sdk.effects import Effect
+from canvas_sdk.effects.note.note import Note as NoteEffect
+from canvas_sdk.effects.simple_api import JSONResponse, Response
+from canvas_sdk.handlers.simple_api import APIKeyAuthMixin, SimpleAPI, api
+from canvas_sdk.v1.data.note import Note, CurrentNoteStateEvent
+
+from charting_api_examples.util import get_note_from_path_params, note_not_found_response
+
+class NoteAPI(APIKeyAuthMixin, SimpleAPI):
+    PREFIX = "/notes"
+
+    """
+    GET /plugin-io/api/charting_api_examples/notes/
+    Headers: "Authorization <your value for 'simpleapi-api-key'>"
+    """
+    @api.get("/")
+    def index(self) -> list[Response | Effect]:
+        notes = Note.objects.select_related('patient', 'provider', 'note_type_version').order_by("dbid")
```

Comment:
When I asked on the Thursday call we had together, I was told you couldnt do select_releated in a plugin? So it is possible? I think it would be worth having a guide about best query practices in plugins for our users. How to fetch only specific elements from a query. Its like we use Django but you can't do everything Django can do since it is limited by sandbox imports, so sometimes it is frustrating.